app/Locale/bs_BA/translations.php

```diff
@@ -648,7 +648,7 @@
     'All tasks' => 'Sve zadatke',
     'Only for tasks assigned to me' => 'Samo za zadatke na kojima sam izvršilac',
     'Only for tasks created by me' => 'Samo za zadatke koje sam ja napravio',
-    'Only for tasks created by me and assigned to me' => 'Samo za zadatke koje sam ja napravio i na kojima sam izvršilac',
+    'Only for tasks created by me tasks and assigned to me' => 'Samo za zadatke koje sam ja napravio i na kojima sam izvršilac',
```

Comment:
The new translation key is `Only for tasks created by me and tasks assigned to me`, there is a typo here, `tasks` should be after the `and`.